build/wix/Localization/mixxx_en-us.wxl

```diff
@@ -43,5 +44,65 @@
    <String Id="FeatureManualDescription">The Mixxx user manual in PDF form, also available at http://mixxx.org/manual/</String>
    <String Id="FeatureTranslationsTitle">Translations</String>
    <String Id="FeatureTranslationsDescription">Resource files that allow Mixxx to display in alternate languages, other than US english</String>
- 
+
+   <!-- Bootstrapper -->
+   <String Id="Caption">[WixBundleName] Setup</String>
+   <String Id="Title">[WixBundleName]</String>
+   <String Id="ConfirmCancelMessage">Are you sure you want to cancel?</String>
+   <String Id="ExecuteUpgradeRelatedBundleMessage">Previous version</String>
+   <String Id="HelpHeader">Setup Help</String>
+   <String Id="HelpText">/install | /repair | /uninstall | /layout [directory] - installs, repairs, uninstalls or
```

Comment:
*Nits found while translating:*
Spare space after ``-``
``/quiet -  ``
Spare space before ``-`` 
``/norestart   -``

I´d write in full ``UI`` -> ``user interface``